tests/test_socket_server.py

```diff
@@ -139,13 +70,31 @@ async def assert_configure(
                 }
             ]
         }
-        await self.write(
-            command=common.Command.CONFIGURE,
-            parameters={common.Key.CONFIGURATION: configuration},
+        await client.write_json(
+            data={
+                common.JsonKeys.COMMAND: common.Command.CONFIGURE,
+                common.JsonKeys.PARAMETERS: {common.Key.CONFIGURATION: configuration},
+            }
```

Comment:
Please consider some syntactic sugar for this -- perhaps a subclass of Client or a utility function. (If its really only needed in this one unit test then a function or method here would suffice, but surely it needed more widely?).

Reply:
I worry about all the places you have to repeat this dict. But it works. Maybe a future improvement.